minerl/env/_multiagent.py

```diff
@@ -270,6 +273,11 @@ def step(self, actions) -> Tuple[dict, dict, bool, dict]:
         if not self.done:
             assert STEP_OPTIONS == 0 or STEP_OPTIONS == 2
 
+            # add chat action if there is one
+            if self.next_chat_message:
+                actions["agent_0"]["chat"] = self.next_chat_message
```

Comment:
Bit of confusion here, but does one have to add the Chat handler to environments to be able to use this? If so, add note about this to documentation and/or throw appropriate error here that informs about this.

Ideally I would like to see that all environments have this option on by default, but I guess that'd involve modifying all existing environments (I could do this) :)

Reply:
Yes, if it isn't enabled _some_ error should get thrown. I will look into doing that.

You should only need to modify `SimpleEmbodimentEnvSpec`.

Reply:
Adding doc/tutorial on how to do this would be enough for now (lets leave modifying other envs for later).

Reply:
Did one better, a new tutorial :)